pkgs/tools/nix/nixpkgs-manual-mmdoc/toc.md

```diff
@@ -0,0 +1,164 @@
+# Nixpkgs Manual
+
+Version 23.11pre
+
+* [](#preface)
```

Comment:
Why are all the links empty?

Are these all linked to content thats still TODO?
Maybe they should be commented out?

Reply:
I believe these have the title auto generated based on other instances of this pattern I've seen elsewhere.